pkg/sql/schemachanger/scerrors/errors.go

```diff
@@ -90,3 +90,26 @@ func ConcurrentSchemaChangeDescID(err error) descpb.ID {
 func ConcurrentSchemaChangeError(desc catalog.Descriptor) error {
 	return &concurrentSchemaChangeError{descID: desc.GetID()}
 }
+
+type schemaChangerUserError struct {
+	err error
+}
+
+// SchemaChangerUserError wraps an error as user consumable, which will surface
+// it from the declarative schema changer without any wrapping.
+func SchemaChangerUserError(err error) error {
```

Comment:
I think this can use examples and some more words about what it would normally be wrapped with.